### ISSUE TITLE
Backport workflow failing

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#7042]
Our backport workflow is not working still - need to fix this before next release!

**Comment by SchrodingersGat** [comment]
Yes, I think the issue was that we did not have explicitly allowed the "pull-requests" permission. Fixed in https://github.com/inventree/InvenTree/pull/7051/files

**Comment by matmair** [comment]
Ok that looks good. Seems like we do not need workflow-wide wright scope?
